src/libratbag-data.c

```diff
@@ -207,6 +215,69 @@ init_data_hidpp20(struct ratbag *ratbag,
 	}
 }
 
+static void
+init_data_sinowealth(struct ratbag *ratbag,
+		     GKeyFile *keyfile,
+		     struct ratbag_device_data *data)
```

Comment:
probably just github's rendering, but the indentation looks off here.

Reply:
I'm not sure how to indent it manually, I just used clang-format file from the Linux project, which more or less works fine in libratbag. Is it okay now?

Reply:
> I'm not sure how to indent it manually, 

I.. wait, what? :smile: Just use tabs and then spaces to align the two?

Reply:
I think it's aligned now.
If it doesn't look right in the diff, you can open the file and see that it's correct.

Reply:
ah, right. the `.editorconfig` states how things should expand, pretty sure these days tabs other than 8 spaces is going to look ugly in most projects. Unfortunately, github renders tabs as 4 spaces which makes everything a bit messy. Looks correct now though, thanks!